<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2dHybridWindowControlWidget</class>
 <widget class="QWidget" name="Graph2dHybridWindowControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="iLabel">
       <property name="text">
        <string>I:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="SliderWithValue" name="iSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="jLabel">
       <property name="text">
        <string>J:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="SliderWithValue" name="jSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="kLabel">
       <property name="text">
        <string>K:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="SliderWithValue" name="kSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="indexLabel">
       <property name="text">
        <string>Index:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="SliderWithValue" name="indexSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lineLabel">
       <property name="text">
        <string>Line:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="lineComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="5">
        <widget class="QPushButton" name="csvExportButton">
         <property name="text">
          <string>CSV &amp;Export</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="copyButton">
         <property name="text">
          <string>&amp;Copy</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="axisButton">
         <property name="text">
          <string>Axis &amp;Setting</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="snapshotButton">
         <property name="text">
          <string>S&amp;napshot</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="dataSourceButton">
         <property name="text">
          <string>&amp;Data Source</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="markerButton">
         <property name="text">
          <string>&amp;Marker Setting</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="fontButton">
         <property name="text">
          <string>&amp;Font Setting</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="drawButton">
         <property name="text">
          <string>Draw &amp;Setting</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWithValue</class>
   <extends>QSlider</extends>
   <header>guibase/widget/sliderwithvalue.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
